history/titles/00_k_wildhammer.txt

```diff
@@ -141,6 +141,9 @@ d_thorvald = {
 	573.1.1={
 		holder=50018
 	}
+	591.9.1={
+		holder=0
```

Comment:
0 can't be a holder.

Reply:
This can work for duchy tier titles and above. Setting the value to 0 destroys the title. No errors in the log.

Reply:
> This can work for duchy tier titles and above. Setting the value to 0 destroys the title. No errors in the log.

Yes, sorry, missed it.